src/nilql/nilql.py

```diff
@@ -735,9 +735,14 @@ def encrypt(
         shares = []
         aggregate = bytes(len(buffer))
         for _ in range(len(key['cluster']['nodes']) - 1):
-            mask = _random_bytes(len(buffer))
+            if len(buffer) > 64:
```

Comment:
maybe add a comment here like:
`If the plaintext length is more than the length of the seed, use the `seed` to generate the `mask`, otherwise, generate it directly.` Or something along these lines